react-backend/client/src/shared/components/form/base.js

```diff
@@ -89,6 +92,9 @@ export const SecondaryButton = Button.extend`
 export const PrimaryButtonNoFill = PrimaryButton.extend`
   background: transparent;
   border-color: ${colors.highlight_primary};
+  &:hover{
+    background:transparent
```

Comment:
Is this necessary? It's also transparent in unhovered state.

Reply:
Yes, it's extended from `PrimaryButton` which was adding a hover colour.